filetest.txt

```diff
@@ -0,0 +1,125 @@
+Attack(
```

Comment:
I don't think you're actually running a test on this file. You need to update the pytest files.